### PR TITLE
[ES] Clean up some beatmap articles

### PR DESCRIPTION
**Pull request by agatemosu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

`SKIP_OUTDATED_CHECK` missing parenthesis in history of osu! 2007

### CODE REVIEW

**Comment by cl8n** [comment]
was this reviewed by someone off-site? i see that box was originally checked

**Review by UnknownOsu — CHANGES_REQUESTED** [review]

File: wiki/History_of_osu!/2007/es.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ A finales de Julio, el sistema de menú fue cambiado para añadir un poco de est
...
+El menú principal fue actualizado con un nuevo diseño, y el editor recibió un montón de nuevas funcionalidades, incluyendo un soporte para copiar y pegar. Se agregaron la pestaña de menú y el menú contextual, lo que hace que las funciones del editor sean más accesibles que antes; los deslizadores fueron más ampliamente compatibles con [tics](/wiki/Gameplay/Hit_object/Slider/Slider_tick) presentes; [apilamiento](/wiki/Beatmapping/Mapping_techniques/Stack) de los [círculos](/wiki/Gameplay/Hit_object/Hit_circle) estaban presentes; las puntuaciones locales más altas recibieron un mejor soporte; y el formato de archivo `.osu` se revisó y renovó por completo, pero incluyó la conversión de archivos más antiguos al nuevo formato.
```
Comment (UnknownOsu) [inline wiki/History_of_osu!/2007/es.md:35]:
```suggestion
El menú principal fue actualizado con un nuevo diseño, y el editor recibió un montón de nuevas funcionalidades, incluyendo un soporte para copiar y pegar. Se agregaron la pestaña de menú y el menú contextual, lo que hace que las funciones del editor sean más accesibles que antes; los sliders fueron más ampliamente compatibles con [tics](/wiki/Gameplay/Hit_object/Slider/Slider_tick) presentes; [apilamiento](/wiki/Beatmapping/Mapping_techniques/Stack) de los [círculos](/wiki/Gameplay/Hit_object/Hit_circle) estaban presentes; las puntuaciones locales más altas recibieron un mejor soporte; y el formato de archivo `.osu` se revisó y renovó por completo, pero incluyó la conversión de archivos más antiguos al nuevo formato.
```

File: wiki/osu!supporter/es.md [hunk 2]
```diff
@@ -7,10 +7,10 @@ tags:
   - etiqueta de osu!supporter
 ---
 
-![Imagen promocional de osu!supporter](img/supporter-eyecatch-ES.jpg)
-
 # osu!supporter
 
+![Imagen promocional de osu!supporter](img/supporter-eyecatch-ES.jpg)
+
 **osu!supporter** (o la ***etiqueta osu!supporter***) es un título temporal dado a los jugadores que apoyan a osu!, de manera voluntaria, con una donación monetaria para ayudarlo a crecer. Los supporters reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el título está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en el juego, rendimiento o puntuación, y nunca lo hará.
```
Comment (UnknownOsu) [inline wiki/osu!supporter/es.md:14]:
```suggestion
**osu!supporter** (o la ***etiqueta de osu!supporter***) es un título temporal dado a los jugadores que apoyan a osu!, de manera voluntaria, con una donación monetaria para ayudarlo a crecer. Los supporters reciben un puñado de beneficios cosméticos y técnicos, la mayoría solo se presentan cuando el título está presente. Las características extras de osu!supporter no dan ningún tipo de ventaja en el juego, rendimiento o puntuación, y nunca lo hará.
```

File: wiki/osu!supporter/es.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ tags:
...
 
 La etiqueta osu!supporter podrá ser obtenida desde la [página de la tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag).
```
Comment (UnknownOsu) [inline wiki/osu!supporter/es.md:16]:
```suggestion
La etiqueta de osu!supporter podrá ser obtenida desde la [página de la tienda de osu!supporter](https://osu.ppy.sh/store/products/supporter-tag).
```

**Review by cl8n — APPROVED** [review]

**Pull request merged**